site/static/robots.txt

```diff
@@ -0,0 +1,8 @@
+User-Agent: *
+Disallow: /cgi-bin/
+Disallow: /&utm_source
+Disallow: /&utm_medium
+Disallow: /&utm_campaign
+Disallow: /?utm_
+
+Sitemap: https://developers.mattermost.com/sitemap.xml
```

Comment:
Non blocking idea: Does it make sense to generate this file based on the configured SiteURL instead of having a static one? This is easily possible with Hugo: https://gohugo.io/templates/robots/

Reply:
My opinion is that it should not be auto-generated since the SEO team can generate these based on the specific needs they are looking for. Having the configured SiteURL pull into the Sitemap reference only adds the benefit of if the site is moved to a different URL, but comes at the cost of the robots.txt file needing to be audited for changes that an auto-generated system might make without warning (should the robots engine change in a future update).